src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformationSummary/UpdatePaymenInformationSummaryTitle/index.js

```diff
@@ -0,0 +1,34 @@
+import { useIntl } from 'react-intl';
+
+const getTitle = () => {
+  return {
+    smallTitle: 'updatePaymentInformationSuccess.title',
+    largeTitle: 'updatePaymentInformationSuccess.unlock_account_title',
+  };
+};
```

Comment:
this code is reused? if it's not reused, maybe it's not necessary! What do you think?

Reply:
Yes, this method is reused for the different status of the reprocess. Depending the status shows different messages.